Note to self: maybe do a local test run next time.

```
FAIL: Swift(linux-x86_64) :: ModuleInterface/distributed-actor.swift (4775 of 18113)
******************** TEST 'Swift(linux-x86_64) :: ModuleInterface/distributed-actor.swift' FAILED ********************
Script:
--
: 'RUN: at line 1';   rm -rf "/home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp" && mkdir -p "/home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp"
: 'RUN: at line 2';   split-file /home/build-user/swift/test/ModuleInterface/distributed-actor.swift /home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp
: 'RUN: at line 4';   /home/build-user/build/buildbot_linux/swift-linux-x86_64/bin/swift-frontend -target x86_64-unknown-linux-gnu  -module-cache-path /home/build-user/build/buildbot_linux/swift-linux-x86_64/swift-test-results/x86_64-unknown-linux-gnu/clang-module-cache -swift-version 4  -define-availability 'SwiftStdlib 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999' -define-availability 'SwiftStdlib 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2' -define-availability 'SwiftStdlib 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0' -define-availability 'SwiftStdlib 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4' -define-availability 'SwiftStdlib 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0' -define-availability 'SwiftStdlib 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5' -define-availability 'SwiftStdlib 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0' -define-availability 'SwiftStdlib 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4' -define-availability 'SwiftStdlib 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0' -define-availability 'SwiftStdlib 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4' -define-availability 'SwiftStdlib 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0' -define-availability 'SwiftStdlib 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1' -define-availability 'SwiftStdlib 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0' -define-availability 'SwiftStdlib 6.1:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999'  -typo-correction-limit 10  -emit-module -module-name Library      -swift-version 5 -enable-library-evolution      -o /home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp/Library.swiftmodule      -emit-module-interface-path /home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp/Library.swiftinterface      /home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp/Library.swift
: 'RUN: at line 11';   /usr/bin/python3.8 /home/build-user/swift/utils/PathSanitizingFileCheck --allow-unused-prefixes --sanitize BUILD_DIR=/home/build-user/build/buildbot_linux/swift-linux-x86_64 --sanitize SOURCE_DIR=/home/build-user/swift --use-filecheck /home/build-user/build/buildbot_linux/llvm-linux-x86_64/bin/FileCheck   /home/build-user/swift/test/ModuleInterface/distributed-actor.swift < /home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp/Library.swiftinterface
: 'RUN: at line 14';   /home/build-user/build/buildbot_linux/swift-linux-x86_64/bin/swift-frontend -target x86_64-unknown-linux-gnu  -module-cache-path /home/build-user/build/buildbot_linux/swift-linux-x86_64/swift-test-results/x86_64-unknown-linux-gnu/clang-module-cache -swift-version 4  -define-availability 'SwiftStdlib 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999' -define-availability 'SwiftStdlib 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2' -define-availability 'SwiftStdlib 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0' -define-availability 'SwiftStdlib 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4' -define-availability 'SwiftStdlib 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0' -define-availability 'SwiftStdlib 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5' -define-availability 'SwiftStdlib 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0' -define-availability 'SwiftStdlib 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4' -define-availability 'SwiftStdlib 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0' -define-availability 'SwiftStdlib 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4' -define-availability 'SwiftStdlib 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0' -define-availability 'SwiftStdlib 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1' -define-availability 'SwiftStdlib 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0' -define-availability 'SwiftStdlib 6.1:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999'  -typo-correction-limit 10  -typecheck -module-name Client      -swift-version 5      /home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp/Client.swift -I/home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp
: 'RUN: at line 19';   rm /home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp/Library.swiftmodule
: 'RUN: at line 20';   /home/build-user/build/buildbot_linux/swift-linux-x86_64/bin/swift-frontend -target x86_64-unknown-linux-gnu  -module-cache-path /home/build-user/build/buildbot_linux/swift-linux-x86_64/swift-test-results/x86_64-unknown-linux-gnu/clang-module-cache -swift-version 4  -define-availability 'SwiftStdlib 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999' -define-availability 'SwiftStdlib 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2' -define-availability 'SwiftStdlib 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0' -define-availability 'SwiftStdlib 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4' -define-availability 'SwiftStdlib 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0' -define-availability 'SwiftStdlib 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5' -define-availability 'SwiftStdlib 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0' -define-availability 'SwiftStdlib 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4' -define-availability 'SwiftStdlib 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0' -define-availability 'SwiftStdlib 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4' -define-availability 'SwiftStdlib 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0' -define-availability 'SwiftStdlib 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1' -define-availability 'SwiftStdlib 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0' -define-availability 'SwiftStdlib 6.1:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999'  -typo-correction-limit 10  -typecheck -module-name Client      -swift-version 5      /home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp/Client.swift -I/home/build-user/build/buildbot_linux/swift-linux-x86_64/test-linux-x86_64/ModuleInterface/Output/distributed-actor.swift.tmp
--
Exit Code: 1

Command Output (stderr):
--
/home/build-user/swift/test/ModuleInterface/distributed-actor.swift:70:11: error: CHECK: expected string not found in input
// CHECK: @available(iOS 9999, tvOS 9999, watchOS 9999, visionOS 9999, macOS 9999, *)
          ^
<stdin>:42:46: note: scanning from here
 public init(actorSystem system: ActorSystem)
                                             ^
<stdin>:43:2: note: possible intended match here
 @available(iOS 18.0, tvOS 18.0, watchOS 11.0, visionOS 2.0, macOS 15.0, *)
 ^

Input file: <stdin>
Check file: /home/build-user/swift/test/ModuleInterface/distributed-actor.swift

-dump-input=help explains the following input dump.

Input was:
<<<<<<
            .
            .
            .
           37:  public typealias SerializationRequirement = any Swift.Decodable & Swift.Encodable 
           38:  deinit 
           39:  nonisolated public var hashValue: Swift.Int { 
           40:  get 
           41:  } 
           42:  public init(actorSystem system: ActorSystem) 
check:70'0                                                  X error: no match found
           43:  @available(iOS 18.0, tvOS 18.0, watchOS 11.0, visionOS 2.0, macOS 15.0, *) 
check:70'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
check:70'1      ?                                                                           possible intended match
           44:  @_semantics("defaultActor") nonisolated final public var unownedExecutor: _Concurrency.UnownedSerialExecutor { 
check:70'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           45:  get 
check:70'0     ~~~~~
           46:  } 
check:70'0     ~~~
           47: } 
check:70'0     ~~
           48: @available(macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0, *) 
check:70'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            .
            .
            .
>>>>>>

--

********************
```

--- a/include/swift/AST/RuntimeVersions.def
+++ b/include/swift/AST/RuntimeVersions.def
@@ -138,12 +138,19 @@ RUNTIME_VERSION(
   PLATFORM(xrOS, (1, 0, 0))
 )
 
+END_MAJOR_VERSION(5)
+
+MAJOR_VERSION(6)
+
 RUNTIME_VERSION(
-  (5, 11),
-  FUTURE
+  (6, 0),
+  PLATFORM(macOS,   (15, 0, 0))
+  PLATFORM(iOS,     (18, 0, 0))
+  PLATFORM(watchOS, (11, 0, 0))
+  PLATFORM(xrOS, (2, 0, 0))
 )
 
-END_MAJOR_VERSION(5)
+END_MAJOR_VERSION(6)
 
 // .......................................................................... //
 

--- a/include/swift/AST/RuntimeVersions.def
+++ b/include/swift/AST/RuntimeVersions.def
@@ -150,6 +150,11 @@ RUNTIME_VERSION(
   PLATFORM(xrOS, (2, 0, 0))
 )
 
+RUNTIME_VERSION(
+  (6, 1),
+  FUTURE
+)
+
 END_MAJOR_VERSION(6)
 
 // .......................................................................... //

--- a/stdlib/public/ClangOverlays/float.swift.gyb
+++ b/stdlib/public/ClangOverlays/float.swift.gyb
@@ -25,7 +25,7 @@ public let FLT_RADIX = Double.radix
 //  significand bit, but Swift does not. Neither is really right or wrong.
 @available(swift, deprecated: 3.0, message: "Please use '${type}.significandBitCount + 1'.")
 @available(macOS 10.9, iOS 7.0, watchOS 2.0, tvOS 9.0, *)
-@_originallyDefinedIn(module: "Darwin", macOS 9999, iOS 9999, watchOS 9999, tvOS 9999)
+@_originallyDefinedIn(module: "Darwin", macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0)
 public let ${prefix}_MANT_DIG = ${type}.significandBitCount + 1
 
 //  Where does the 1 come from? C models floating-point numbers as having a
@@ -34,32 +34,32 @@ public let ${prefix}_MANT_DIG = ${type}.significandBitCount + 1
 //  as well.
 @available(swift, deprecated: 3.0, message: "Please use '${type}.greatestFiniteMagnitude.exponent + 1'.")
 @available(macOS 10.9, iOS 7.0, watchOS 2.0, tvOS 9.0, *)
-@_originallyDefinedIn(module: "Darwin", macOS 9999, iOS 9999, watchOS 9999, tvOS 9999)
+@_originallyDefinedIn(module: "Darwin", macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0)
 public let ${prefix}_MAX_EXP = ${type}.greatestFiniteMagnitude.exponent + 1
 
 @available(swift, deprecated: 3.0, message: "Please use '${type}.leastNormalMagnitude.exponent + 1'.")
 @available(macOS 10.9, iOS 7.0, watchOS 2.0, tvOS 9.0, *)
-@_originallyDefinedIn(module: "Darwin", macOS 9999, iOS 9999, watchOS 9999, tvOS 9999)
+@_originallyDefinedIn(module: "Darwin", macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0)
 public let ${prefix}_MIN_EXP = ${type}.leastNormalMagnitude.exponent + 1
 
 @available(swift, deprecated: 3.0, message: "Please use '${type}.greatestFiniteMagnitude' or '.greatestFiniteMagnitude'.")
 @available(macOS 10.9, iOS 7.0, watchOS 2.0, tvOS 9.0, *)
-@_originallyDefinedIn(module: "Darwin", macOS 9999, iOS 9999, watchOS 9999, tvOS 9999)
+@_originallyDefinedIn(module: "Darwin", macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0)
 public let ${prefix}_MAX = ${type}.greatestFiniteMagnitude
 
 @available(swift, deprecated: 3.0, message: "Please use '${type}.ulpOfOne' or '.ulpOfOne'.")
 @available(macOS 10.9, iOS 7.0, watchOS 2.0, tvOS 9.0, *)
-@_originallyDefinedIn(module: "Darwin", macOS 9999, iOS 9999, watchOS 9999, tvOS 9999)
+@_originallyDefinedIn(module: "Darwin", macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0)
 public let ${prefix}_EPSILON = ${type}.ulpOfOne
 
 @available(swift, deprecated: 3.0, message: "Please use '${type}.leastNormalMagnitude' or '.leastNormalMagnitude'.")
 @available(macOS 10.9, iOS 7.0, watchOS 2.0, tvOS 9.0, *)
-@_originallyDefinedIn(module: "Darwin", macOS 9999, iOS 9999, watchOS 9999, tvOS 9999)
+@_originallyDefinedIn(module: "Darwin", macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0)
 public let ${prefix}_MIN = ${type}.leastNormalMagnitude
 
 @available(swift, deprecated: 3.0, message: "Please use '${type}.leastNonzeroMagnitude' or '.leastNonzeroMagnitude'.")
 @available(macOS 10.9, iOS 7.0, watchOS 2.0, tvOS 9.0, *)
-@_originallyDefinedIn(module: "Darwin", macOS 9999, iOS 9999, watchOS 9999, tvOS 9999)
+@_originallyDefinedIn(module: "Darwin", macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0)
 public let ${prefix}_TRUE_MIN = ${type}.leastNonzeroMagnitude
 
 % if type == "Float80":

--- a/stdlib/public/core/ErrorType.swift
+++ b/stdlib/public/core/ErrorType.swift
@@ -195,7 +195,7 @@ func _willThrowTypedImpl<E: Error>(_ error: E)
 @_alwaysEmitIntoClient
 @_silgen_name("swift_willThrowTyped")
 public func _willThrowTyped<E: Error>(_ error: E) {
-  if #available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999, *) {
+  if #available(macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0, *) {
     _willThrowTypedImpl(error)
   }
 }

--- a/utils/availability-macros.def
+++ b/utils/availability-macros.def
@@ -35,7 +35,7 @@ SwiftStdlib 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0
 SwiftStdlib 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4
 SwiftStdlib 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0
 SwiftStdlib 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1
-SwiftStdlib 6.0:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999
+SwiftStdlib 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0
 # TODO: Also update ASTContext::getSwift510Availability when needed
 # TODO: Also update ASTContext::getSwift60Availability when needed
 

